README.md

```diff
@@ -39,16 +39,16 @@ Install the Google CAS Issuer CRDs in `config/crd`. These manifests use kustomiz
 kubectl apply -k config/crd
 ```
 
-Examine the ClusterRole and ClusterRolebinding in `config/rbac/role.yaml` and
-`config/rbac/role_binding.yaml`. By default, these give the default Kubernetes service
-account in the cert-manager namespace all the necessary permissions. Customise these to your needs.
+Examine the ClusterRole and ClusterRolebinding in `config/rbac/role.yaml` and `config/rbac/role_binding.yaml`. By default, these give the default Kubernetes service account in the cert-manager namespace all the necessary permissions. Customise these to your needs.
```

Comment:
I realized that `give the default Kubernetes service account` should now be `give the ksa-google-cas-issuer Kubernetes service account`.
Also we should have another line `kubectl apply -f config/rbac/serviceaccount.yaml` here.
Could we please add that here 😅 ?

Reply:
Sure, this is fixed in b67444a and 4619932